src/components/UI/search-box/search-box.module.scss

```diff
@@ -1,11 +1,33 @@
-.searchContainer {
+.searchbox_container {
   display: flex;
-  align-items: center;
-  padding: 6px;
+  width: clamp(200px, 80%, 500px);
+  height: 40px;
+  border-radius: 10px;
+  box-shadow: 5px 5px 10px grey;
+  margin: 20px auto;
+  top: 90px;
+  position: sticky;
 }
-.searchBox {
-  float: left;
-  padding: 4px;
-  border: 1px solid rgba(0, 0, 0, 0.397);
-  border-radius: 5px;
+
+.button {
+  background-color: #382bd0;
+  color: white;
+  font-size: 12px;
+  width: 100px;
+  cursor: pointer;
+  border-radius: inherit;
+  border-top-left-radius: 0;
+  border-bottom-left-radius: 0;
+  border: none;
+}
+
+.searchbox {
+  background: #e4e5e4;
+  border: none;
+  border-radius: inherit;
+  border-top-right-radius: 0;
+  border-bottom-right-radius: 0;
+  flex-grow: 1;
+  padding-left: 20px;
+  text-overflow: ellipsis;
```

Comment:
Why are we not using rem?

Reply:
in the whole project we haven't used that, so I proceeded with px only whatever was mentioned in the Figma design